packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -497,7 +552,7 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
         }
         if (op.mode === 'activate') {
             this.shell.activateWidget(widget.id);
-        } else if (op.mode === 'reveal') {
+        } else {
```

Comment:
I don't think it is correct, there is a 3rd option `open` which means that a widget should be attached but not revealed. Now it will reveal as well.